tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/AddUser.java

```diff
@@ -123,7 +124,7 @@ private AddUserResponse doAddNewUser(AddUserRequest request, CookieSession sessi
     private String getNewUserLocales(String requestLocales) {
         requestLocales = LocaleNormalizer.normalizeQuietly(requestLocales);
         if (requestLocales.isEmpty()) {
-            return "mul";
+            return LocaleNames.MUL;
```

Comment:
This is a bug I'm working on separately. Can you remove it from this ticket?

Reply:
I guess its OK for now

mul is also used for the sandbox (which is the meaning here) . That meaning should be ideally encapsulated separately

Reply:
maybe this is at least a step in that direction? Would they both still be named "mul"?

Reply:
yes, they are both named mul